### PR TITLE
TruSTAR bug

### PR DESCRIPTION
**Pull request by dantavori**
## Status
In Hold (no instance, waiting for customer approval for bug fix)

## Related Issues
fixes: [https://github.com/demisto/etc/issues/19193](https://github.com/demisto/etc/issues/19193)

## Description
1. Packagifying
2. Fixed an issue where the command ***trustar-search-indicator*** returned an incorrect context output

## Does it break backward compatibility?
   - No

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/TruSTAR_Bug/Integrations/TruSTAR/CHANGELOG.md)

### CODE REVIEW

**Comment by kirbles19** [comment]
@dantavori Done.

**Comment by dantavori** [comment]
@yaakovi

File: Integrations/TruSTAR/TruSTAR.py [hunk 1]
```diff
@@ -0,0 +1,752 @@
...
+                                                 demisto.params().get('url_threshold'), ('URL',), create_url_ec))
+
+    elif demisto.command() == 'domain':
+        demisto.results(generic_search_indicator(demisto.args().get('domain'), demisto.params().get('domain_threshold'),
+                                                 ('Domain', 'URL',), create_domain_ec))
+
+except Exception, e:
+    LOG(e.message)
+    LOG.print_log()
+    raise
```
Comment (yaakovi) [inline Integrations/TruSTAR/TruSTAR.py]: as long as we are here:
```
except Exception as e:
    return_error(e)
```

File: Integrations/TruSTAR/TruSTAR.py [hunk 2]
```diff
@@ -0,0 +1,752 @@
...
+        current_report['updated'] = normalize_time(current_report['updated'])
+        current_report['created'] = normalize_time(current_report['created'])
+        current_report['timeBegan'] = normalize_time(current_report['timeBegan'])
+        reports.append(current_report)
+        report_context.append({
+            'reportTitle': current_report['title'],
+            'id': current_report['id']
+        })
+        if 'reportBody' in current_report:
+            report_context[0]['reportBody'] = current_report['reportBody']
```
Comment (yaakovi) [inline Integrations/TruSTAR/TruSTAR.py]: why are you appending to the first context and not the one relevant for the current report?

File: Integrations/TruSTAR/TruSTAR.py [hunk 3]
```diff
@@ -0,0 +1,752 @@
...
+import os
+import collections
+from trustar.models.indicator import Indicator
+from trustar.models.page import Page
+
+if not demisto.params()['proxy']:
+    del os.environ['HTTP_PROXY']
+    del os.environ['HTTPS_PROXY']
+    del os.environ['http_proxy']
+    del os.environ['https_proxy']
```
Comment (yaakovi) [inline Integrations/TruSTAR/TruSTAR.py]: replace this with `handle_proxy()`

File: Integrations/TruSTAR/TruSTAR.py [hunk 4]
```diff
@@ -746,7 +741,7 @@ def get_enclaves():
         demisto.results(generic_search_indicator(demisto.args().get('domain'), demisto.params().get('domain_threshold'),
                                                  ('Domain', 'URL',), create_domain_ec))
 
-except Exception, e:
-    LOG(e.message)
+except Exception as e:
+    LOG(str(e))
```
Comment (yaakovi) [inline Integrations/TruSTAR/TruSTAR.py]: no need to call `LOG` and `LOG.print_log`. 
`return_error` already doing that.

**Pull request merged**